src/reference/asciidoc/kafka.adoc

```diff
@@ -1073,6 +1074,18 @@ See its javadocs for more information.
 A retry adapter is not provided for any of the batch <<message-listeners, message listeners>> because the framework has no knowledge of where, in a batch, the failure occurred.
 Users wishing retry capabilities, when using a batch listener, are advised to use a `RetryTemplate` within the listener itself.
 
+[[stateful-retry]]
+===== Stateful Retry
+
+It is important to understand that the retry discussed above suspends the consumer thread (if a `BackOffPolicy` is used); there is no interaction with the broker during retry.
```

Comment:
`there is no interaction with the broker during retry`

During retry, the background heartbeat thread is alive and can interact with the broker. Does `there is no poll() call to the broker during retry` sound better?